src/hooks/dialog/dialog.stories.tsx

```diff
@@ -98,6 +98,8 @@ export const Prompt: Story = {
               "Annuleren",
               args.onConfirm,
               args.onCancel,
+              undefined,
```

Comment:
The public interface doesn't doesn't to change bacause of this refactor right? It just implements the behaviour of useFormDialog for code reuse?